src/app/app.component.ts

```diff
@@ -3,7 +3,11 @@ import { RouterLink, RouterLinkActive, RouterOutlet } from '@angular/router';
 import { TaskCreateComponent } from './task-create/task-create.component';
 import { TaskListComponent } from './task-list/task-list.component';
 
-
+interface navItem {
+  title: string;
```

Comment:
create a separate model for this like navbar.model.ts file where you declare these interfaces/types and export them so other components can use them